src/actors/client/ActorProxyFactory.interface.ts

```diff
@@ -1,8 +0,0 @@
-import Class from "../../types/Class";
-import ActorId from "../ActorId";
-import ActorProxy from "./ActorProxy";
-import AbstractActor from "../runtime/AbstractActor";
-
-export default interface ActorProxyFactoryInterface {
```

Comment:
breaking change?

Reply:
There is a ProxyBuilder now that implements Proxy itself. This was old deprecated code and was not functional